src/axom/core/tests/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ set(core_serial_tests
     core_map.hpp
     core_flatmap.hpp
     core_memory_management.hpp
+    # core_numeric_array.cpp
```

Comment:
Why are the tests commented out?

Reply:
a temporary change I didn't mean to commit.  Fixing.